source/tutorials/install.txt

```diff
@@ -0,0 +1,24 @@
+.. guide::
+   title: Installing MongoDB Build Tools
+   author: MongoDB Documentation Team
+   type: Deep Dive 
+   level: intermediate 
+   product_version: 4.0
+   result_description:
+     In this guide you will learn how to install the MongoDB Build Tools. Note that this guide only applies to MacOS officially.
+   time: 45
+   prerequisites:
+     - access to git
+   check_your_environment:
+     - check to make sure you have git installed
+   procedure:
+     .. include:: /includes/steps/install.rst
+   summary:
+     If you have successfully completed this guide, you have 
+     installed the MongoDB Build Tools.
+   whats_next:
+     Congratulations! Now that you've installed the Build Tools, you are ready to start building your site.
+   seealso:
+     - `check out the guides site for examples <http://docs.mongodb.com/guides>`__.
+     - :doc:`Tabbed Content Tutorial </tutorials/tabbed-content>`
+     - :doc:`Create a Docstools-Enabled Repo </tutorials/repo>`
```

Comment:
I think this should be first.  I also think the other ones aren't super relevant here so it might dilute the importance of this link.